app/pages/ServiceDiscoveryResults/SearchResults/SearchResults.scss

```diff
@@ -69,4 +104,15 @@
     white-space: nowrap;
     overflow: scroll;
   }
-}
\ No newline at end of file
+
+  @media screen and (max-width: 1120px) {
+    flex-direction: column;
```

Comment:
You should probably have this selector reset the `width` back to `inherit`, since otherwise it'll continue to have the 250px width from the non-media-query selector. Without doing this, you end up with the phone numbers being truncated:

<img width="434" alt="Screen Shot 2020-07-14 at 12 10 21 AM" src="https://user-images.githubusercontent.com/1002748/87395591-77795d80-c566-11ea-9448-efba96a91a31.png">